src/main/resources/application.properties

```diff
@@ -32,14 +28,14 @@ rabbitmq.user: myuser
 rabbitmq.password: myuser
 # Check configuration documentation for using jasypt encrypted passswords
```

Comment:
I have no idea why it's placed like that. The jasypt password is not set in properties though since it would expose the "secret" or at least make it easier to find. Perhaps just remove those lines?

Reply:
Removed this one but the mongodb one I kept since when I merged a lot of comments were added...